lib/modules/manager/bundler/artifacts.ts

```diff
@@ -43,8 +43,24 @@ function buildBundleHostVariable(hostRule: HostRule): Record<string, string> {
   };
 }
 
+const resolvedPkgRegex = regEx(
+  /(?<pkg>\S+)(?:\s*\([^)]+\)\s*)? was resolved to/g
+);
+
+function getResolvedPackages(input: string): string[] {
+  const result: string[] = [];
+  let resolveMatchGroups = resolvedPkgRegex.exec(input)?.groups;
+  while (resolveMatchGroups) {
+    const { pkg } = resolveMatchGroups;
+    result.push(pkg);
```

Comment:
```suggestion
    if (result.includes(pkg)) {
        logger.debug('pkg already found in result');
        break;
    }
    result.push(pkg);
```

Reply:
Added uniqueness instead. I'm not sure early break would protect from infinite loop.